Rhino.Licensing/LicenseGenerator.cs

```diff
@@ -43,7 +43,7 @@ public string GenerateFloatingLicense(string name, string publicKey)
                 var publicKeyEl = doc.CreateElement("license-server-public-key");
                 license.AppendChild(publicKeyEl);
                 publicKeyEl.InnerText = publicKey;
-                
+

```

Comment:
Can you extract this trim change to its own `maint` commit along with the other styling only changes?

Reply:
Done!